Add Active Learning Paper

bstienen opened the pull request:
This pull request adds a paper on Active Learning in HEP to the papers.md and the HEPML.bib files.

bstienen committed:
Add paper to papers.md file

bstienen committed:
Add active learning paper to bibtex

matthewfeickert commented on papers.md:
```diff
@@ -18,6 +18,8 @@ theory](https://inspirehep.net/record/1785309)," arXiv:2003.06413 [hep-lat]. (Ma
 
 - M. Borisyak, N. Kazeev, "[Machine Learning on data with sPlot background subtraction](https://inspirehep.net/record/1737258)," arXiv:1905.11719 [cs.LG]. (May 28, 2019)
 
+- S. Caron, T. Heskes, S. Otten and B. Stienen, "[Constraining the Parameters of High-Dimensional Models with Active Learning](https://arxiv.org/abs/1905.08628)," Eur. Phys. J. C 79, 944 (2019), arXiv:1905.08628 [cs.LG]. (May 19, 2019)
```
```suggestion
- S. Caron, T. Heskes, S. Otten and B. Stienen, "[Constraining the Parameters of High-Dimensional Models with Active Learning](https://inspirehep.net/literature/1735784)," Eur. Phys. J. C 79, 944 (2019), arXiv:1905.08628 [cs.LG]. (May 19, 2019)
```

matthewfeickert replied:
We prefer to have the links go to INSPIRE rather than directly the arXiv, as INSPIRE is a good hub to get to all forms of the publication.

matthewfeickert reviewed (changes requested):
Thanks very much for the nice PR, @bstienen! We really appreciate contributions, and you've made this one super easy. If you can just change the URL for the paper I think this one is ready to go in. :+1:

bstienen committed:
Change arXiv link to INSPIRE-HEP

arXiv link for Active Learning paper changed to the corresponding INSPIRE-HEP link.

Co-Authored-By: Matthew Feickert <matthew.feickert@cern.ch>

bstienen commented:
Ah, good catch! I added your suggestion to the pull request.

matthewfeickert reviewed (approved):
Oh, I forgot to also ask that you just add the citation to the `.tex` document (which is how I generate the citation to add to the `paper.md`). I can just add this though and then merge it in. Thanks again for the contribution!

matthewfeickert committed:
Add citation to .tex file
